engines/scumm/detection.cpp

```diff
@@ -1120,7 +1120,7 @@ Common::Error ScummMetaEngine::createInstance(OSystem *syst, Engine **engine) co
 		md5Warning = _("Your game version appears to be unknown. If this is *NOT* a fan-modified\n"
 		               "version (in particular, not a fan-made translation), please, report the\n"
 		               "following data to the ScummVM team along with name of the game you tried\n"
-		               "to add and its version/language/etc.:\n");
```

Comment:
“add” should still be in the sentence. Best written the same way as suggested for the same content in `advancedDetector.cpp`: “…to add *and* its version, …”